akka-projection-kafka/src/test/resources/application.conf

```diff
@@ -0,0 +1,21 @@
+akka {
+  loggers = ["akka.event.slf4j.Slf4jLogger"]
+  loglevel = "DEBUG"
+  logging-filter = "akka.event.slf4j.Slf4jLoggingFilter"
+  logger-startup-timeout = 15s
+
+  actor {
+    debug {
+      lifecycle = off
+      receive = off
+    }
+  }
+
+  test {
+    single-expect-default = 20s
```

Comment:
What's the reason for such long default timeout? Is it some initialization? Can we have a long timeout only for the initialization timeout?

Reply:
I actually only used this during development debug.  I'll use the default or a smaller value.

Reply:
Actually, we may not need this configuration file at all.  Removing it.